### DISCUSSION TITLE
Community challenges

### DISCUSSION THREAD

**Discussion by raphodn** [openfoodfacts/open-prices#509] [General]
# Idea

The idea of periodic challenges emerged during the OFF Days 2024.

These kinds of challenges find good success in the OpenStreetMap community, with either [MapRoulette](https://maproulette.org/) or the [monthly projects (link in french)](https://www.openstreetmap.fr/category/contribute/projet-du-mois/) organized by OSM France.

In order to inform contributors and to track progress, we could imagine a dedicated page.

## In the app

https://prices.openfoodfacts.org/challenges

## Examples

see below

## Reuse-driven

Find ideas that will nudge some users to reuse the data, or help an NGO gather data.

## Related issues

- https://github.com/openfoodfacts/open-prices-frontend/issues/934
- https://github.com/openfoodfacts/open-prices/discussions/1281

**Comment by raphodn** [comment]
@DamienMAYAUX mentioned a "price collection" challenge to help compare your co-op's prices with surrounding shops
- restrict on a list of shops to go to
- restrict on a timespan (start & end date)
- possibily restrict on a list of products or categories

And have a dashboard that lists user contribution, products still missing, possibility to export data.

**Comment by raphodn** [comment]
## First challenge

Hazelnut spreads

**Reply by raphodn** [reply]
### Analysis ?

- Analyzing how price evolve geographically (for the same distributor) or depending on the distributor would be interesting in my opinion
- If Prices data can be linked to OFF data we could determine if the % of hazelnut in the spread is linked to the price
- Can spreads be clustered by price ? Like is there a cheap cluster vs a high end cluster ? Are there outliers with regards to ingredients in these clusters (e.g. a product with palm oil and lots of sugar in the high price cluster) ? We could make a cool graph with logos
- Most prices are in France, with Candlemas (La Chandeleur) landing in the middle of the challenge. The tradition is to eat crêpes to celebrate. I've noticed stores pushing crêpes related products, such as spreads, and was wondering if we could observe an effect on prices (Discounts vs increase to match the demand ?)

**Reply by raphodn** [reply]
ChatGPT 4

1. Price Distribution & Trends
Histogram of Prices: Show the distribution of hazelnut spread prices across different regions and stores
Price Evolution Over Time: analyze price changes over time of the challenge
Outliers & Anomalies: Identify unusually high or low prices and potential reasons (e.g., promotions, incorrect entries)
2. Geographic Price Variations
Regional Price Differences: Compare hazelnut spread prices by country, city, or even neighborhood (if location data is detailed enough)
Heatmap of Prices: Visualize price differences geographically using a heatmap
Store-Type Comparison: Compare prices in supermarkets, convenience stores, discount stores, etc.
3. Store & Brand Analysis
Price Variation by Retailer: Identify which stores offer the lowest and highest prices
Private label vs. hazelnut spread: using available data on alternative spreads, compare pricing strategies
Size & Packaging Effects: Look at the price per kilogram across different hazelnut spread sizes (e.g., 200g, 500g, 1kg)
4. Inflation & Economic Indicators (not Challenge-specific, as it's too short a timeperiod)
Compare with Inflation Rates: If you have historical data, see if Nutella prices follow general inflation trends.
Correlation with Economic Events: Check if Nutella prices respond to factors like supply chain disruptions, cocoa/hazelnut price fluctuations, or economic downturns.
5. Citizen Behavior Insights (not Challenge-specific, as it's too short a timeperiod)
Price Sensitivity: Do people report more prices when they see a discount?
Effect of Promotions: Identify if clusters of low prices indicate promotional campaigns
Peak Reporting Times: Check if price submissions are seasonal (e.g., around holidays)
6. Data Quality & Contribution Analysis
Data Completeness: Are there missing data points (e.g., missing stores, regions, etc.)?
User Contribution Patterns: Identify top contributors and potential duplicate entries
Verification Rate: If multiple contributors submit the same store’s price, how often do they match?

**Comment by raphodn** [comment]
## Challenge ideas

- milk vs milk substitutes (OFF days 2024)
- find top products in OFF
- average basket products
- products for ongoing research projects ?

**Reply by TTalex** [reply]
* Treasure hunt for specific products / brands (partnership with producers / stores ?)
* Match some current news (Example with Trump administration and the price of eggs https://edition.cnn.com/2025/01/28/economy/trump-inflation-price-promises/index.html)

**Reply by Freso** [reply]
it's summer! ice cream (incl. vegan/non-dairy ones) and/or sun screens?

**Reply by Freso** [reply]
Do you remember who suggested “milk vs milk substitutes” in 2024 @raphodn ?

**Comment by serpico** [comment]
Based on my own observations, something seems to be going on in the raclette's market ( branded products on sale in a designated zone of the supermarket week in, week out and this week even the _marque premier prix_ is on sale at a 25% discount ( anti-gaspi )....

So a community challenge could be an opportunity to get a bigger sample.

I also take the opportunity to say I'd love to get some deeper insights into why some price cycles happen ( I'm no economist ) and I feel open-prices lay the foundation to allow that but so far I don't know who to turn to...( blogger, social media,...). 

I've read about mvark in a recent OFF's blog post ( apparently he played/plays with stats on the nutritional side of things ) so I was thinking about asking him for his take...

**Comment by raphodn** [comment]
### Top products sold

![image](https://github.com/user-attachments/assets/396f67c5-9e3d-4a8f-8889-3d5fdf8a285f)

**Reply by raphodn** [reply]
```
Product Name,Barcode,Price
"Eau de source plate CRISTALINE","3274080001005","1.14"
"Eau de source état naturel Cristaline","3274080005003","0.19"
"COCA-COLA 1,25","5449000267412","1.65"
"COCA-COLA 1,75","5000112611861","2.03"
"RED BULL 25cl","9002490205973","1.09"
"COCA-COLA 6*33cl","5449000283900","3.69"
"VOLVIC 6*1,5l","3057640257858","3.24"
"CRISTALINE 12*50cl","3254381204312","1.92"
"Hepar 6x1l","7613035974692",""
"CRISTALINE 50cl","",""
"Vitel 6*1,5l","7613036249935","2.94"
"Coca-cola 1l","5449000054227",""
"HEINEKEN","3119780268696","1.19"
"MONSTER ENERGY","5060335632302","1.60"
"Président DOUX 250g","3155251205500","2.75"
"Evian 6*1,5l","3068320120263","3.72"
"Tropical OASIS","3124480191182","2.09"
"PERRIER 6*1l","7613035833289","3.90"
"Mozzarella GALBANI","8000430138719","1.25"
"Ice Tea pêche LIPTON","5900497611503",""
"BAVARIA Original 8.6","8714800038669","1.45"
"MONT ROUCOUS 6x1l","3257971091316","3.12"
"Harrys 100% mie","3228857000852","1.75"
"Pepsi max 1,5l","",""
"SOIGNON 200g","3523230028431","2.25"
"DESPERADOS Original 50cl","3119780266401","2.81"
"COCA-COLA Zero 1,75l","5000112611878","1.79"
"Pains au lait PASQUIER 350g","3256540000698","1.49"
"Camembert PRESIDENT","3228021170039","1.89"
"HERTA 170g","3154230809302","2.29"
```

**Reply by TTalex** [reply]
_Related suggestion by Julia on slack:_

Maybe a challenge could be top 10 food items? If we want to compare the cost of living around the world, we need it. If not ten, even a top five could reveal surprising differences.
For example, TOP 10 food items for comparing cost of living:

1. Bread (1 loaf)
2. Milk (1 liter)
3. Eggs (10 pcs)
4. Rice (1 kg)
5. Chicken breast (1 kg)
6. Potatoes (1 kg)
7. Apples (1 kg)
8. Cheese (1 kg)
9. Vegetable oil (1 liter)
10. Sugar (1 kg)

Specific quantities like 1 kg or 1 liter - this makes the data consistent and comparable. Most of the countries don't have these products in the DB.
The challenge would be completed once a person uploads all 10 specified products. The OP would then review the submission or update it live.

**Reply by TTalex** [reply]
_Related suggestion by g123k on slack:_

The challenges are great for getting quick pricing on a category. But once they’re done, we have the data, but what’s the actual goal?

I’m wondering if it would be worth having something running in parallel—like a “shopping basket” of comparable goods, similar to what newspapers track (like this example from Le Parisien).This would let us compare products across different retailers in the short term, while also seeing how prices evolve in the long term.

<img width="622" height="883" alt="image" src="https://github.com/user-attachments/assets/13fb6efc-53e1-417e-8f5f-7b7ca092742f" />

**Reply by raphodn** [reply]
dedicated discussion on basket/index tracking: https://github.com/openfoodfacts/open-prices/discussions/1281

**Comment by raphodn** [comment]
April : add photos & prices of "green" products (packaging :green_book:)

**Comment by raphodn** [comment]
Non-food products (cosmetics, DIY stuff) to add new products to the OxF DB

**Reply by raphodn** [reply]
Would require a way to create new products easily from Open Prices

**Reply by raphodn** [reply]
And tricky to validate prices for "unknown" products

**Comment by raphodn** [comment]
New upcoming challenge: Summer !
:sun_with_face: ice creams & sun screens :sun_with_face:

**Reply by Freso** [reply]
:hand_over_mouth: i wonder where that idea came from :innocent:

**Comment by raphodn** [comment]
September : "back to school" :books:

**Comment by raphodn** [comment]
October : Halloweeeen :jack_o_lantern:

**Comment by serpico** [comment]
Products which last price dates back 12 month ( give or take ), then depending on how many products were fetched another filtering round to make it an achievable goal ( considering the current overall contribution trend on OP )

**Reply by TTalex** [reply]
One way to have this kind of "update", would be to redo similar challenges a year later. It would be easier to for contributors, rather than looking for specific products (Although I like the "treasure hunt" idea of struggling a bit to find a specific product)

**Comment by Sebleouf** [comment]
The latest season-related challenges were cool, but sometimes the range of products was too limited.
This is just my personal opinion, but I think it would be more motivating for future challenges to have larger categories that are less culturally specific (not everyone celebrates Christmas or Halloween, and summer doesn't happen at the same time everywhere), in order to include all regions. This will ensure that the maximum number of people can participate, rather than just 9 for Halloween challenge.
For example, we could have a challenge on ‘ready meals’, ‘chocolates’, ‘fish’, ‘bread‘ or ‘sweet drinks’ for food, and alternate challenges between food and non-food products.
Do you agree?

**Reply by TTalex** [reply]
Yep, fully agree. There's a balance to find between too specific challenges (halloween) and too large ones (Back to school included all snacks).
One benefit of specific challenges is that it also incites product creation, so we get low numbers on Open Prices, but quite a few products were created on Open Food Facts and Open Product facts because of them.
Wider challenges result in higher numbers, but most come from contributors that didn't know a challenge was going on. They just happen to add product that fit in the category. So I wouldn't count them as real challenge contributors.

What seems to be a good balance is to have a widespread category (usually food) and a bit more niche one at the same time. The egg one might be a good example (Regular eggs + chocolate eggs).

**Comment by Freso** [comment]
“Potatoes” ([`en:Potatoes and their products`](https://world.openfoodfacts.org/facets/categories/potatoes-and-their-products))

Potato is a seasonal food for March (at least in northern hemisphere), and re: https://github.com/openfoodfacts/open-prices/discussions/509#discussioncomment-15463271

**Reply by Freso** [reply]
Hiding this as resolved because this was done march/april 2026: https://prices.openfoodfacts.org/challenges/11

**Comment by raphodn** [comment]
New idea for a new kind of challenge: gather as much receipts as you can :)
- even if it's not yours
- so we won't be able to add the corresponding products
- but it'll be useful to map to the related OSM shop
- and grow the "receipt" dataset when we'll want to do more automation there

**Comment by Freso** [comment]
Would be neat if we could make challenges based on labels ([`organic`](https://world.openbeautyfacts.org/facets/labels/organic), [`no palm oil`](https://world.openbeautyfacts.org/facets/labels/no-palm-oil), …) or brands or even ingredients ([`palm oil`](https://world.openbeautyfacts.org/facets/ingredients/palm-oil)? [`perfume`](https://world.openbeautyfacts.org/facets/ingredients/perfume)?).

**Reply by raphodn** [reply]
super interesting idea !

**Reply by raphodn** [reply]
or nova or nutriscore

**Reply by raphodn** [reply]
World Vegan Month is November: https://www.vegansociety.com/get-involved/world-vegan-month
